File: api/TWTodos/tables/Grupo.cs
public class Grupo
{
    public int IdGrupo { get; set; }
    public string Nome { get; set; } = string.Empty;
    public DateTime DataCriacao { get; set; }
    public string Descricao { get; set; } = string.Empty;
    public ICollection<Membro> Membros { get; set; } = new List<Membro>();
}

File: CotuTalk_Program/app/src/main/java/com/example/cotutalk_program/AcessoAPI/data/Grupo.kt
package com.example.cotutalk_program.AcessoAPI.data
import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import com.google.gson.annotations.SerializedName
import org.threeten.bp.LocalDateTime

@Parcelize
data class Grupo(
    @SerializedName("idGrupo")
    val IdGrupo: Int,
    @SerializedName("nome")
    val Nome: String,
    @SerializedName("dataCriacao")
    val DataCriacao: LocalDateTime,
    @SerializedName("descricao")
    val Descricao: String,
): Parcelable
